### DISCUSSION TITLE
APBS calculations give almost same results for different ionic strengths

### DISCUSSION THREAD

**Discussion by srdjan125** [Electrostatics/apbs#153] [Q&A]
Hello,

I would like to ask a question regarding my calculations. 

  I am trying to calculate binding energy between 2 proteins according to this tutorial: https://www.ics.uci.edu/~dock/manuals/apbs/html/tutorial/x307.html . 
  Parameters are same like in this tutorial, except fglen and cglen and dime which were obtained by pysize.py script (or by online server PDB2PQR).

Proteins are also moved away from each other, so I obtain plot proteinA-proteinB energy vs their distance at several different salt concentrations.
 I have tried it for one small and one large protein (monoclonal antibody : PDB code 3npo).
  In case of small protein, I get that energy curves are decreasing while salt concentration is increasing (similar to plots I would get with simple Debye-Huckel model).
   However, in case of large protein with irregular shape (monoclonal antibody) I receive results that curve is only different at 0.01M salt concentration. For all other concentrations (0.16M, ...., 0.25, ...., 05 , .... 1M) I get that plots are basically the same, ie there is no salt screening effect and proteinA-proteinB energy curves are basically the same at all those concentrations.

   I have tried to perform calculations with linear PB vs non-linear PB (lpbe vs npbe), I have tried mdh vs sdh boundary conditions however results still remain the same.
   I have also tried to perform calculations on same protein, but with another PDB structure, and still the same results.

  This can mean either
a) I am making some mistake or missing some important detail which leads to getting wrong results
b) Results are correct, there is huge solvation/desolvation penalty which is signifficantly larger than screened Couloumb energy, so salt screening effect is lost.

   I would appreciate if someone could help me how to debug this problem and figure out if these results are correct and trustable. 

  Greetings.

**Comment by sobolevnrm** [comment]
Hello --

I suspect your hypothesis (b) is correct.  However, there is an easy way to test this:  increase the solute dielectric `pdie` and see if the salt effect becomes more pronounced.

Thank you

**Comment by srdjan125** [comment]
Hello,

Thank you for answer. I have tried pdie 10 or pdie 20 and I got little bit different curves depending on salt concentrations, they are still similar but at least not the same like in previous case (pdie 2).
  For me these results seems reasonable, but I have to make sure that calculations are accurate.
    During each calculations APBS gives message "Debye Huckel length 30A/12A/3A" etc, I even tried calculations with surface and charges chgm spl4 srfm spl4 and still I obtain the same trends.


________________________________
